src/pkg/logger/example_logger_test.go

```diff
@@ -0,0 +1,120 @@
+package logger_test
+
+import (
+	"context"
+
+	"github.com/alcionai/clues"
+	"github.com/alcionai/corso/src/pkg/logger"
+)
+
+// ---------------------------------------------------------------------------
+// mock helpers
+// ---------------------------------------------------------------------------
+
+const (
+	loglevel = "info"
+	logfile  = "stderr"
+	itemID   = "item_id"
+)
+
+var err error
+
+// ---------------------------------------------------------------------------
+// examples
+// ---------------------------------------------------------------------------
+
+// ExampleSeed showcases seeding a logger into the context.
+func Example_seed() {
```

Comment:
Let's add `logging.md` to https://github.com/alcionai/corso/tree/main/website/docs/developers with just a link to these files.

Reply:
done.